### PR TITLE
Add Formentry widget

### PR DESCRIPTION
**Pull request by enyachoke**

### CODE REVIEW

File: src/formentry/formentry.tsx [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  const formEntryParcel = () => System.import("@hackathon/formentry-widget");
+  return (
+    <div>
+      {" "}
+      <Parcel config={formEntryParcel} />
+    </div>
+  );
+}
+
+type FormeEntryProps = {};
```
Comment (joeldenning) [inline src/formentry/formentry.tsx]: typo: should be `FormEntryProps`

File: src/dashboard-widgets/dashboard-widgets.tsx [hunk 2]
```diff
@@ -3,16 +3,16 @@ import { css } from "@emotion/core";
...
+  () => import("../formentry/forms-parcel").then(m => m.default)
 ];
 
 export default function DashboardWidgets(props: DashboardWidgetsProps) {
   return (
     <div
       css={css`
         margin: 76px auto 0 auto;
-        width: 500px;
+        width: 100%;
```
Comment (joeldenning) [inline src/dashboard-widgets/dashboard-widgets.tsx]: Why did you change this?

Comment (enyachoke) [reply src/dashboard-widgets/dashboard-widgets.tsx]: Was trying something wasn’t  supposed  to push that.
